bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChartView.java

```diff
@@ -592,7 +592,9 @@ private void onSelectionChanged(final ISelection selection) {
                            new SelectionChartXSliderPosition(//
                                  _tourChart,
                                  tourDataSelection.getLeftSliderValueIndex(),
-                                 tourDataSelection.getRightSliderValueIndex()));
+                                 tourDataSelection.getRightSliderValueIndex(),
+                                 _prefStore.getDefaultBoolean(
+                                       ITourbookPreferences.TOGGLE_STATE_SELECT_INBETWEEN_TIME_SLICES)));
```

Comment:
This code is used in many places, would it not be easier to put it in the `SelectionChartXSliderPosition` constructor or read this value in the tour editor where the selection is performed?

Reply:
>>would it not be easier to put it in the SelectionChartXSliderPosition
I can't do that as otherwise it creates a circular dependencies
![image](https://user-images.githubusercontent.com/25235665/86273101-f5b02a00-bb8c-11ea-858e-929504bcb0e9.png)


>>or read this value in the tour editor where the selection is performed?

That's what I did in my first [original PR](https://github.com/wolfgang-ch/mytourbook/pull/192) but you told me not to

![image](https://user-images.githubusercontent.com/25235665/86273190-1bd5ca00-bb8d-11ea-91ae-925c594be935.png)

Reply:
Addressed in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/192)